swarm.sh

```diff
@@ -73,7 +81,9 @@ main() {
   elif [[ "${ACTION}" == "down" ]]; then
     log info "Cares only has down for its dependencies"
   elif [[ "${ACTION}" == "destroy" ]]; then
-    log info "Cares only has destroy for its dependencies"
+    docker::service_destroy cares-clickhouse-config-importer
```

Comment:
Your PR says remove configs and services when destroy, but I don't see any configs being removed here

Reply:
This PR includes two fixes: 
* I updated the function "generate_service_configs" in Platform to accept another argument "config_label" to add a label so that the clickhouse configs will be correctly labeled and then removed when destroy. 
* The fhir config importer helper was failing and it wasn't removed after destroying the stack. So I added those service_destroy to make sure whatever the service is failed or not it should be removed.